testsuite/tests/regression/pr9326/gc_set.ml

```diff
@@ -0,0 +1,25 @@
+(* TEST
+*)
+
+open Gc
+
+let min_heap_sz = 524288 (* 512k *)
+let maj_heap_inc = 4194304 (* 4M *)
+
+let _ =
+  let g1 = Gc.get() in
+  Gc.set { g1 with minor_heap_size = min_heap_sz;
+                   major_heap_increment = maj_heap_inc };
+  let g2 = Gc.get() in
```

Comment:
One could make the test to fail next time new fields are added and force people to update it by deconstructing either g1 or g2 and relying on exhaustive pattern in record.
```ocaml
let {minor_heap_size; ...} = Gc.get() in
```

Reply:
Good idea.  Done.